packages/terra-search-field/src/SearchField.jsx

```diff
@@ -258,6 +275,7 @@ class SearchField extends React.Component {
             ariaLabel={inputText}
             aria-disabled={isDisabled}
             onKeyDown={this.handleKeyDown}
+            onInput={this.handleInput}
```

Comment:
We may want to consider adding onInput as first class prop to the terra-form-input/Field so we don't ever lose track of this. Some teams may use dictation for regular inputs.

Reply:
I agree with this, but don't think it should be part of this PR. Will log an issue.